src/index.ts

```diff
@@ -175,14 +177,19 @@ export class CruxClient {
         // log.debug(`Encryption key:`, this._getEncryptionKey())
 
         // Setting up the default modules as fallbacks
-        this._storage =  this._options.storage || new storage.LocalStorage();
+        if (this._options.privateKey) {
+            this._keyPair =  utils.getKeyPairFromPrivKey(this._options.privateKey);
```

Comment:
Is there any difference between
`if (this._keyPair)` and `if (this._authenticated)`? 

If there's no logical difference we can get rid of `this._authenticated`

Reply:
as this._keyPair is removed once payIDClaim is set, there is this need for a new property.